client/src/js/app.js

```diff
@@ -322,13 +322,39 @@ function bhimaConfig($stateProvider, $urlRouterProvider, $urlMatcherFactoryProvi
   })
 
   /* debtors routes */
-
-  .state('debtors/groups', {
-    url : '/debtors/groups',
+  .state('debtorGroups', {
+    url : '/debtors/groups/:uuid',
+    // url : '/debtors/groups',
+    abstract : true,
+    params : {
+      uuid : { squash : true, value : null }
+    },
+    // url : '/notathing',
     controller : 'DebtorGroupController as GroupCtrl',
     templateUrl: 'partials/debtors/groups.html'
   })
 
+    .state('debtorGroups.list', {
+      url : '',
+      templateUrl : 'partials/debtors/groups.list.html'
+    })
+
+    .state('debtorGroups.create', {
+
+      // setting the URL as simply create mathces as a :uuid - there should be a way to set orders
```

Comment:
For future note, this is a `@todo` that we can explore .  This route implementation is great though!